mlflow/ml-package-versions.yml

```diff
@@ -275,6 +275,15 @@ statsmodels:
 spark:
   package_info:
     pip_release: "pyspark"
+    install_dev: |
+      temp_dir=$(mktemp -d)
+      git clone --depth 1 https://github.com/apache/spark $temp_dir
+      cd $temp_dir
+      export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=1g"
+      ./build/mvn -DskipTests --no-transfer-progress clean package
+      cd python
+      python setup.py sdist
+      pip install dist/*.tar.gz
```

Comment:
Is it possible to cache the build result ? and set cache expire time to be 1 day.
the compiling will took a long time.

Reply:
I think it's possible. Let me try.

Reply:
offline discussion: We'll address this in a follow-up.